docusaurus/docs/create-a-plugin/develop-a-plugin/panel-properties.md

```diff
@@ -0,0 +1,106 @@
+---
+id: subscribe-events
+title: Subscribe to Grafana events
+description: Work with the event bus to subscribe to Grafana application events
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - panel
+  - properties
+  - eventbus
+  - event bus
+  - subscribe
+  - events
+sidebar_position: 5
+---
+
+# Subscribe to Grafana events
+
+If you’re building a [panel plugin](../../introduction/plugin-types-usage.md#panel-visualization-plugins), Grafana already provides the data you need to configure and render your panel. If not, you can usually retrieve the information you need from any of the runtime services, like `BackendSrv` and `TemplateSrv`.
+
+However, in some cases you may want your plugin to react to changes that occur outside of your plugin. For example, you may want your plugin to react when the user zooms in or out of another panel. In this guide, you’ll learn how to make your plugin react to events in Grafana.
```

Comment:
non-blocking - this doesn't seem a particularly strong example, it would be good if there was a more concrete use case here including both the event and the change a user would likely want to make in their plugin as a result of the event.

Reply:
modified example

Reply:
The example remains the same, but this is non blocking